scanner/Makefile

```diff
@@ -56,12 +56,20 @@ ifeq ($(shell uname -s),Darwin)
 endif
 GOOS := $(HOST_OS)
 
-GO_BUILD_FLAGS = CGO_ENABLED=$(CGO_ENABLED) GOOS=$(GOOS) GOARCH=$(GOARCH)
+# TODO(ROX-27054): Remove the redundant strictfipsruntime option if one is found to be so
+ifeq ($(GOTAGS),)
+	GOTAGS := "strictfipsruntime"
+else
+	# Preserve existing GOTAGS and append "strictfipsruntime"
+	GOTAGS := $(GOTAGS),"strictfipsruntime"
+endif
```

Comment:
I'm not sure it works when assignments are indented. I thought Make wants its own syntax to be unindented. But my actual request: can you please find a way to also make this one-liner? In my opinion, it does not deserve five lines. Perhaps, `blah := $(if <condition>, <then>, <else>)` can do the job here?